ros2cli/ros2cli/node/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# TODO(mikaelarguedas) revisit this once it's specified
+HIDDEN_NODE_PREFIX = '_'
```

Comment:
+1 in consolidating the declaration of HIDDEN_NODE_PREFIX

Reply:
Imo the consolidation should not happen here in `ros2cli` (I assume that is why you kept the TODO comment) but go into `rclpy`. Similar to `HIDDEN_TOPIC_PREFIX`.